packages/ui5-task-zipper/lib/zipper.js

```diff
@@ -95,7 +116,11 @@ module.exports = async function ({ log, workspace, dependencies, options, taskUt
 					zipEntries.push(resourcePath);
 					return resource.getBuffer().then((buffer) => {
 						isDebug && log.info(`Adding ${resource.getPath()} to archive.`);
-						zip.addBuffer(buffer, resourcePath); // Replace first forward slash at the start of the path
+						if (relativePaths && resourcePath.includes("manifest.json")) {
```

Comment:
Sorry, missed that one - includes is wrong here - as it should be matching, right? But maybe you want to handle all included manifest.json files - but then you need to pass also the `resourcePath` to the method `absoluteToRelativePaths` to put the file into the proper location again? Otherwise all manifest.json files e.g. manifest.json, reuse-component/manifest.json will overwrite the *manifest.json* in the root. But you could also return the modified buffer from the function and then move zip.addBuffer outside of the else branch so that it runs also for the modification case.

```js
if (relativePaths && resourcePath.includes("manifest.json")) {
  buffer = absoluteToRelativePaths(buffer);
}
zip.addBuffer(buffer, resourcePath); // Replace first forward slash at the start of the path
```

Reply:
Good catch. Let's only modify the `manifest.json`. I will change it to `resourcePath === "manifest.json"`.